### DISCUSSION TITLE
VisualStudio2022 智能代码提示和补全

### DISCUSSION THREAD

**Discussion by PranksterJ** [BYJRK/DotNet-Discussions#129] [C# 技术交流]
有哪位大神知道 #vs上这种代码提示是如何实现的吗（C#和xaml代码都能实现）？
![1](https://github.com/BYJRK/DotNet-Discussions/assets/102715817/53eb206b-a8e6-4960-a3a4-005b5be5b1e4)
![2](https://github.com/BYJRK/DotNet-Discussions/assets/102715817/5560346a-2693-45ad-ac52-cd142b326df2)

**Comment by ccq1** [comment]
首先你要下载一个copilot扩展，然后登录你的github账号。就可以了。学生用户可以在github进行学生认证，copilot免费

**Reply by PranksterJ** [reply]
收到 十分感谢
